src/main/java/com/compilercharisma/chameleonbusinessstudio/config/SecurityConfiguration.java

```diff
@@ -16,8 +16,6 @@ public SecurityWebFilterChain securityWebFilterChain(
             ServerHttpSecurity security) throws Exception {
         return security
                 .authorizeExchange()
-                .pathMatchers("/api/users/**")
```

Comment:
This is the line we need to add when we test endpoints on Postman, right? Irc this was you're work around the auth wall

Reply:
Yeah, that's so we can actually like see what's going on with the endpoints without having to authenticate ourselves, we can alternatively authenticate ourselves using Postman but that would be more complicated.